MSBot/src/BotConfig.ts

```diff
@@ -241,14 +227,14 @@ export class BotConfig implements IBotConfig {
             }
 
             this.internal.secretValidated = true;
-        } catch{
-            throw new Error("You are attempting to perform an operation which needs access to the secret and --secret is incorrect.");
+        } catch {
```

Comment:
Is this invalid syntax? TypeScript requires an identifier for the exception, AFAIK. e.g.: `catch (e)`

Reply:
[optional catch binding](https://github.com/tc39/proposal-optional-catch-binding) is a stage 3 proposal that typescript should transpile for us. I think we're safe here.